app/controllers/declaration/CusCodeController.scala

```diff
@@ -78,3 +78,10 @@ class CusCodeController @Inject()(
       controllers.declaration.routes.TaricCodeSummaryController.displayPage(_, itemId)
     }
 }
+
+object CusCodeController {
```

Comment:
IMO this object should be placed under `controllers/utils`

Reply:
Ok, I've created a new generic `ExportsDecModelHelper` controller util class to hold this method.